app/views/candidate_mailer/_reasons_for_rejection_intro.text.erb

```diff
@@ -1,5 +1,9 @@
 <% unless @application_choice.rejected_by_default? %>
 
-  <%= @course.provider.name %> has decided not to make you an offer to study <%= @course.name_and_code %>. They've given feedback to explain this decision.
+  <%= @course.provider.name %> has reviewed your application for <%= @course.name_and_code %>.
+
+  On this occasion, they’ve decided not to make an offer.
```

Comment:
Minor suggestion – I wonder if this would feel slightly clearer with a "you"?

```suggestion
  On this occasion, they’ve decided not to make you an offer.
```

Reply:
Another option:

```suggestion
  On this occasion, they’ve decided not to offer you a place.
```

Reply:
I feel like both of those are clearer but less 'soft'

Reply:
heh yea. I was trying to come up with it way so it feels less personal. but maybe it's impossible to avoid 'you'

Reply:
![Screenshot 2022-03-10 at 17 18 54](https://user-images.githubusercontent.com/56349171/157718978-e0c7888e-de72-436b-810f-0ec6131ce956.png)

Reply:
have done some re-wording. let me know if you think this reads ok!

Reply:
I feel like `they’ve decided not to offer you a place on this course` is 'softer' but i'm not the expert 😅 ✍️

Reply:
I know what u mean! It's just sooo many words - bit of a tongue twister 'decided not to...'